NotificationSystem.Tests/EmailSenderTests.cs

```diff
@@ -0,0 +1,155 @@
+using Moq;
+using NotificationSystem.Notifications;
+using NotificationSystem.Services;
+using NUnit.Framework;
+using SendGrid;
+using SendGrid.Helpers.Mail;
+using System;
+using System.Linq;
+using System.Threading;
+
+namespace NotificationSystem.Tests
+{
+    public class EmailSenderTests
+    {
+        [Test]
+        public void ParsePayload_ValidBody_EmailNotification()
+        {
+            EmailNotification notification = new EmailNotification("test@example.com", "plain text content");
+            string payload = Newtonsoft.Json.JsonConvert.SerializeObject(notification);
+            EmailSender emailSender = new EmailSender(null, "test");
+
+            emailSender.ParsePayload(payload);
+
+            Assert.AreEqual(notification.RecipientEmail, emailSender.Notification.RecipientEmail);
+        }
+
+        [Test]
+        public void ParsePayload_InvalidBody_Exception()
+        {
+            //arange
```

Comment:
According to our wiki: `Insert one space between the comment delimiter (//) and the comment text, as shown in the following example.` ([https://github.com/DigitalExcellence/dex-backend/wiki/Coding-guidelines#commenting-conventions](https://github.com/DigitalExcellence/dex-backend/wiki/Coding-guidelines#commenting-conventions)).

Is there a reason why you didn't follow this rule here? Otherwise, I would suggest to follow this guideline.